api/proto/disperser/disperser.proto

```diff
@@ -78,7 +78,7 @@ message DisperseBlobRequest {
 	// Clients can define one SecurityParams per quorum, and specify multiple quorums.
 	// The disperser will ensure that the encoded blobs for each quorum are all processed
 	// within the same batch.
-	repeated SecurityParams security_params = 2;
+	bytes quorums = 2;
```

Comment:
Since it's in byte, would be better to specify what's the encoding format are expected from users

Reply:
Would it be better to make this a Uint32? I'm having second thoughts about using `byte` now.

Reply:
If it's an integer typed, then users have no need to decide how to code the data into byte. But in that case,`uint8`?

Reply:
I actually like the typed field which is more clear and enforced. `byte` can be filled with anything and users can more easily shoot themselves in the foot.

Reply:
Grpc doesn't have the uint8 type, only Uint32 and Uint64. I think using Uint32 should be fine. GRPC apparently uses varint encoding, so it's still efficient.

Reply:
Sounds good. The efficiency loss should be quite small, whereas the benefit of type checking at interface is considerable.